src/Compilers/CSharp/Portable/Symbols/Source/ModifierUtils.cs

```diff
@@ -62,6 +62,27 @@ internal static class ModifierUtils
             return result;
         }
 
+        internal static void ReportDefaultInterfaceImplementationModifiers(
+            LanguageVersion availableVersion,
+            LanguageVersion requiredVersion,
+            DeclarationModifiers modifiers,
+            DeclarationModifiers defaultInterfaceImplementationModifiers,
+            Location errorLocation,
+            DiagnosticBag diagnostics)
+        {
+            DeclarationModifiers errorModifiers = modifiers & defaultInterfaceImplementationModifiers;
+            while (errorModifiers != DeclarationModifiers.None)
+            {
+                DeclarationModifiers oneError = errorModifiers & ~(errorModifiers - 1);
+                Debug.Assert(oneError != DeclarationModifiers.None);
+                errorModifiers = errorModifiers & ~oneError;
+                diagnostics.Add(ErrorCode.ERR_DefaultInterfaceImplementationModifier, errorLocation,
+                                ConvertSingleModifierToSyntaxText(oneError),
+                                availableVersion.ToDisplayString(),
```

Comment:
Similarly, consider moving availableVersion.ToDisplayString() to a local outside the loop.